### ISSUE TITLE
Panic when joining channels

### ISSUE THREAD

**Issue by Chronophylos** [museun/twitchchat#186]
I am using a slightly modified version of the async-std example.

When I try to join a Channel I get:
```
panicked at 'called `Result::unwrap()` on an `Err` value: CannotParseTag { name: "color", error: InvalidHexString }'
```
The panic originates from here:
https://github.com/museun/twitchchat/blob/ae5de19395e7d555f68cefda70cba288c2290948/src/runner/async_runner.rs#L537

**twitchchat version**: 0.12.0

Might be important: The account I'm using as no color set in chat.

**Comment by Emilgardis** [comment]
https://github.com/museun/twitchchat/blob/ae5de19395e7d555f68cefda70cba288c2290948/src/messages/global_user_state.rs#L77-L85 should check for empty color tag here. Example GLOBALUSERSTATE = `@badge-info=;badges=;color=;display-name=justintv;emote-sets=0,459289,300548757;user-id=1337;user-type= :tmi.twitch.tv GLOBALUSERSTATE\r\n`

**Comment by museun** [comment]
I just published v0.12.1 which fixes this. It was from parsing GLOBALUSERSTATE.
